core/vm/operations_acl.go

```diff
@@ -32,11 +32,14 @@ func makeGasSStoreFunc(clearingRefund uint64) gasFunc {
 		}
 		// Gas sentry honoured, do the actual gas calculation based on the stored value
 		var (
-			y, x    = stack.Back(1), stack.peek()
-			slot    = common.Hash(x.Bytes32())
-			current = evm.StateDB.GetState(contract.Address(), slot)
-			cost    = uint64(0)
+			y, x         = stack.Back(1), stack.peek()
+			slot         = common.Hash(x.Bytes32())
+			current, err = evm.StateDB.GetState(contract.Address(), slot)
```

Comment:
I think its nicer to take this equation out of this collection. Since we use this err.

Reply:
I don't understand. Can you elaborate? We use all of these variables, don't we?

Reply:
I meant for the above line to be moved out of the big var block, looks weird to catch the error after few lines. (the error returned here is caught after few lines below)

Reply:
Ah, gotcha. yeah, I agree

Reply:
Fixed: 4b4a442